ocaml-lsp-server/src/config_data.ml

```diff
@@ -0,0 +1,261 @@
+open Import
+open Import.Json.Conv
+
+module Lens = struct
```

Comment:
It would be nice to rename this module to `Codelens`, so that it's easier to figure out it's purpose and consistent with field name.